tf-psa-crypto/drivers/builtin/src/psa_crypto_ecp.c

```diff
@@ -617,7 +617,36 @@ psa_status_t mbedtls_psa_generate_key_iop_setup(
     return mbedtls_to_psa_error(status);
 }
 
-psa_status_t mbedtls_psa_generate_key_iop_abort(
+psa_status_t mbedtls_psa_ecp_generate_key_iop_complete(
+    mbedtls_psa_generate_key_iop_t *operation,
+    uint8_t *key_output,
+    size_t key_output_size,
+    size_t *key_len)
+{
+    *key_len = 0;
+    psa_status_t status = PSA_ERROR_CORRUPTION_DETECTED;
```

Comment:
```suggestion
    int status = PSA_ERROR_CORRUPTION_DETECTED;
```

Sorry for not catching this before, but the type is not strictly correct. Indeed this is used with `mbedtls_ecp_gen_privkey` and `mbedtls_mpi_write_binary` that return `int` and, also, you are using the converting function `mbedtls_to_psa_error` at the end to change `int`-->`psa_status_t`.

The same holds also for `mbedtls_psa_ecp_generate_key_iop_setup`.